src/docs/release/breaking-changes/rendereditable-layout-before-hit-test.md

```diff
@@ -106,15 +107,16 @@ class FakeEditableTextState extends TextSelectionDelegate {
 
 ## Timeline
 
-This change landed in v1.18.0.
+This change landed in v1.18.
 
 ## References
 
 API documentation:
 * [`RenderEditable`][]
 
 Relevant issue:
-* [Issue 43494: SelectableText.rich used along with TapGestureRecognizer is not working][]
+* [Issue 43494: SelectableText.rich used along with
+   TapGestureRecognizer is not working][]
```

Comment:
I didn't expect this to work!

Reply:
Oh yeah! I try to avoid breaking lines in the square brackets, but it does work!